FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,57 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+
+			switch result {
+			case let .success((data, response)):
+				completion(FeedImageMapper.map(data, from: response))
+			case .failure(_):
```

Comment:
`(_)` is not necessary, it can be removed.

Please remove `(_)`

```suggestion
			case .failure:
```

Reply:
Removing it results in an error **Tuple pattern cannot match values of the non-tuple type 'Error'**. Maybe because we're using `Swift.Result` generic enum instead of a type defined `enum LoadFeedResult`?

Reply:
Please ensure you removed underscore along with braces. It should be exactly as in my suggestion.